docs/UserGuide.md

```diff
@@ -98,6 +98,7 @@ A person can have any number of tags (including 0)
 - Note that Name, Phone Number, Email and Address are compulsory fields. 
 - We believe it is reasonable that a customer or business contacts will need to provide these fields as they are not 
 particularly sensitive as compared to Identification Number etc.
+- Email does not check for `.com` specifically at the end of each email as custom domain names exist. E.g. `@u.nus.edu`
```

Comment:
```suggestion
-The email validation does not check for the presence of a period (.) after the "@" symbol, and it does not verify any specific domain extension. It only ensures that the domain name after the "@" is at least two characters long.
```

Reply:
Also would it be better to move to another section like I have done with the phone numbers? (where i created a new section called future features)